### ISSUE TITLE
Chalet requires repository URLs to end with `.git` while git itself doesn't have such requirement

### ISSUE THREAD

**Issue by thatbakamono** [chalet-org/chalet#156]
Chalet requires repository URLs to end with `.git`:
```
ERRORS: Failed to validate the json file: chalet.json
   An invalid value was found in 'repository': 'https://github.com/fmtlib/fmt'. Expected the pattern '^(?:git|ssh|git\+ssh|https?|git@[\w\-.]+):(\/\/)?(.*?)(\.git)(\/?|#[\w\d\-._]+?)$'
```
while git itself doesn't have such requirement and can do just fine without them as can be seen below:
```
C:\Users\Bakamono\Desktop>git clone https://github.com/chalet-org/chalet
Cloning into 'chalet'...
```
```
C:\Users\Bakamono\Desktop>git clone git@github.com:chalet-org/chalet
Cloning into 'chalet'...
```

**Comment by rewrking** [comment]
Thanks for noticing that. I'll look into relaxing that requirement. In the meantime, it will work with `git@github.com:fmtlib/fmt.git`

**Comment by rewrking** [comment]
Fixed in [v0.4.1](https://github.com/chalet-org/chalet/releases/tag/v0.4.1)
